### ISSUE TITLE
PyFMI integration test fails locally on Windows

### ISSUE THREAD

**Issue by markaren** [NTNU-IHB/PythonFMU#32]
I have never been able to run the PyFMI integration tests locally.
PyFMI installed with pip. Latest FMILibrary.
This is the error produced:

```
Launching pytest with arguments test_integration.py::test_integration_get in C:\Users\LarsIvar\Documents\IdeaProjects\PythonFMU\pythonfmu\tests

============================= test session starts =============================
platform win32 -- Python 3.7.4, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: C:\Users\LarsIvar\Documents\IdeaProjects\PythonFMU, inifile: setup.cfgcollected 1 item

test_integration.py F
pythonfmu\tests\test_integration.py:30 (test_integration_get)
tmp_path = WindowsPath('C:/Users/LarsIvar/AppData/Local/Temp/pytest-of-LarsIvar/pytest-65/test_integration_get0')

    @pytest.mark.integration
    def test_integration_get(tmp_path):
    
        script_file = Path(__file__).parent / DEMO
    
        FmuBuilder.build_FMU(script_file, dest=tmp_path, needsExecutionTool="false")
    
        fmu = tmp_path / "PythonSlave.fmu"
        assert fmu.exists()
>       model = pyfmi.load_fmu(str(fmu))

test_integration.py:40: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src\pyfmi\fmi.pyx:7941: in pyfmi.fmi.load_fmu
    ???
src\pyfmi\fmi.pyx:6488: in pyfmi.fmi.FMUModelCS2.__init__
    ???
src\pyfmi\fmi.pyx:3761: in pyfmi.fmi.FMUModelBase2.__init__
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   TypeError: latin_1_encode() argument 1 must be str, not bytes

src\pyfmi\fmi.pyx:45: TypeError
                                                    [100%]

================================== FAILURES ===================================
____________________________ test_integration_get _____________________________

tmp_path = WindowsPath('C:/Users/LarsIvar/AppData/Local/Temp/pytest-of-LarsIvar/pytest-65/test_integration_get0')

    @pytest.mark.integration
    def test_integration_get(tmp_path):
    
        script_file = Path(__file__).parent / DEMO
    
        FmuBuilder.build_FMU(script_file, dest=tmp_path, needsExecutionTool="false")
    
        fmu = tmp_path / "PythonSlave.fmu"
        assert fmu.exists()
>       model = pyfmi.load_fmu(str(fmu))

test_integration.py:40: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src\pyfmi\fmi.pyx:7941: in pyfmi.fmi.load_fmu
    ???
src\pyfmi\fmi.pyx:6488: in pyfmi.fmi.FMUModelCS2.__init__
    ???
src\pyfmi\fmi.pyx:3761: in pyfmi.fmi.FMUModelBase2.__init__
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   TypeError: latin_1_encode() argument 1 must be str, not bytes

src\pyfmi\fmi.pyx:45: TypeError
============================== 1 failed in 1.73s ==============================
Process finished with exit code 0

```

**Comment by markaren** [comment]
Seems the issue is the PyFMI version. It has been fixed in 2.5.1, but pip only serves 2.5..
https://github.com/modelon-community/PyFMI/issues/10

**Comment by markaren** [comment]
`pip3 install git+https://github.com/modelon-community/PyFMI.git` did the trick!
